client/src/index.js

```diff
@@ -6,14 +6,15 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import AllCharacters from './componenets/AllCharacters';
 import CharacterSearch from './pages/characterSearch/CharacterSearch';
 import PageNotFound from './pages/PageNotFound';
+import CharacterGallery from './pages/charactersGalery/CharacterGallery';
 
```

Comment:
Please use with the React.lazy
const CharacterGallery = React.lazy(() => import('./pages/charactersGalery/CharacterGallery'));
*be noticed that in lazy loading the component must to be export default!*